app/containers/HomePage.js

```diff
@@ -3,9 +3,13 @@
  * @flow
  * @TODO: Use waitForImages plugin to load background images and fade in on load
  */
-import { bindActionCreators } from 'redux';
```

Comment:
What was the reason for removing `bindActionCreators`?

Reply:
It's not necessary as the component is already aware of redux via `connect`, so it can just dispatch the actions directly in `mapDispatchToProps`.

See https://redux.js.org/api/bindactioncreators